packages/balances/src/types/balances.ts

```diff
@@ -255,6 +255,17 @@ export class Balances {
   }
 }
 
+export const getBalanceId = (
+  balance: Pick<
+    BalanceJson,
+    "source" | "subSource" | "address" | "chainId" | "evmNetworkId" | "tokenId"
+  >
+) => {
+  const { source, subSource, address, chainId, evmNetworkId, tokenId } = balance
+  const locationId = chainId !== undefined ? chainId : evmNetworkId
+  return [source, address, locationId, tokenId, subSource].filter(Boolean).join("-")
```

Comment:
maybe this is the right opportunity to use `::` as separator instead of `-` ? 
we have some `-` in chain ids, probably in other fields too, making it hard to parse when necessary

Reply:
It's fine, although afaik the situations where we are parsing the ID are only in getting the 'placeholder balances', which will no longer exist. Are there any other reasons why the ID needs to be parseable?

Reply:
not really, for now, it's just that it feels wrong to use a separator that's also used in the values that compose the id.
same issue in token ids actually